coderd/database/queries/provisionerjobs.sql

```diff
@@ -21,8 +21,13 @@ WHERE
 			nested.started_at IS NULL
 			-- Ensure the caller has the correct provisioner.
 			AND nested.provisioner = ANY(@types :: provisioner_type [ ])
-			-- Ensure the caller satisfies all job tags.
-			AND nested.tags <@ @tags :: jsonb
+			AND CASE
+				-- Special case for untagged provisioners: only match untagged jobs.
+				WHEN nested.tags :: jsonb = '{"scope": "organization", "owner": ""}' :: jsonb
```

Comment:
When we assign @ tags, will it always contain `scope = organization` when untagged and used live?

Reply:
We make sure to call `provisionersdk.MutateTags` before serializing tags, so it will always contain `scope` and `owner` keys.